src/Dash.NET/DashComponents/ComponentBase.fs

```diff
@@ -120,6 +126,13 @@ module ComponentPropTypes =
             Title:string
         }
         static member create label value disabled title = {Label=label; Value=value; Disabled=disabled; Title=title}
+        static member convert this =
```

Comment:
If we want to stop using the camelCase serializer settings, we should add JsonProperty attributes (like [here](https://github.com/plotly/Dash.NET/blob/ef9806918f4e87c50e27c25960f670c01401272b/src/Dash.NET/Callback.fs#L26)) to all records that will not get serialized correctly with the new settings. Another example (which is why we used camelCase serialization can be seen [here](https://github.com/plotly/Dash.NET/pull/24#issuecomment-921761973))

Reply:
Good point - that would be clearer.

Reply:
Just to make sure we're on the same page: The problem with the camel-case field names is with "external" data, i.e. when it is being used to pass data in, e.g.
```fsharp
DataTable.dataTable "memory-table" [
    DataTable.Attr.data [
        {| Country = "Canada" |}
        {| Country = "USA" |}
    ]
]
```
`Country` would be camel-cased and would fail to bind correctly to the header record "Country".